freeipa/src/main/resources/schema/app/20191210120103_CDPCP-1025_add_cooldown_to_usersyncstatus.sql

```diff
@@ -0,0 +1,13 @@
+-- // CDPCP-1025 track the last time usersync is requested
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE usersyncstatus
+    ADD COLUMN IF NOT EXISTS lastfullsyncstarttime bigint,
```

Comment:
Add a comment on the format of this column, i.e., milliseconds from java epoch. This should be UTC I think.

Reply:
added